client/src/telethon/_impl/mtproto/mtp/encrypted.py

```diff
@@ -245,22 +243,54 @@ def _handle_rpc_result(self, message: Message) -> None:
         result = rpc_result.result
 
         msg_id = MsgId(req_msg_id)
-        inner_constructor = struct.unpack_from("<I", result)[0]
+
+        try:
+            inner_constructor = struct.unpack_from("<I", result)[0]
+        except struct.error as e:
+            # If the result is empty, we can't unpack it.
+            # This can happen if the server returns an empty response.
+            logging.exception(e)
```

Comment:
`logging.exception` already knows the exception when called inside an `except` block.

```suggestion
        except struct.error:
            # If the result is empty, we can't unpack it.
            # This can happen if the server returns an empty response.
            logging.exception('failed to unpack inner_constructor')
```